File: mobile/android/app/src/main/java/com/pedidoprime/centraloperacional/model/OperationalMetrics.kt
package com.pedidoprime.centraloperacional.model

data class OperationalMetrics(
    val mesasAbertas: Int = 0,
    val emPreparo: Int = 0,
    val turnoAtual: Double = 0.0
) {
    companion object {
        val ZERO = OperationalMetrics()
    }
}

File: mobile/ios/CentralOperacionalApp/Models/OperationalMetrics.swift
import Foundation

/// Métricas em tempo real da operação (mesas, preparo, faturamento do turno).
/// Espelha o shape retornado pela API (`GET /operacional/metricas`).
struct OperationalMetrics: Decodable, Equatable {
    var mesasAbertas: Int
    var emPreparo: Int
    var turnoAtual: Decimal

    static let zero = OperationalMetrics(mesasAbertas: 0, emPreparo: 0, turnoAtual: 0)

    enum CodingKeys: String, CodingKey {
        case mesasAbertas = "mesas_abertas"
        case emPreparo = "em_preparo"
        case turnoAtual = "turno_atual"
    }
}
